ftl/python/builder.py

```diff
@@ -33,76 +33,106 @@ class Python(builder.JustApp):
     def __init__(self, ctx):
         self.descriptor_files = [_REQUIREMENTS_TXT]
         self.namespace = _PYTHON_NAMESPACE
+        self.ctx = ctx
         super(Python, self).__init__(ctx)
 
     def __enter__(self):
         """Override."""
         return self
 
-    def _generate_overrides(self):
+    def _generate_overrides(self, py_version='python2.7'):
         return metadata.Overrides(
-            creation_time=str(datetime.date.today()) + "T00:00:00Z")
-
-    def CreatePackageBase(self, base_image, cache):
+            creation_time=str(datetime.date.today()) + "T00:00:00Z",
+            env={
+                "VIRTUAL_ENV":
+                "/env",  # not working properly, not sure why
+                "PYTHONUSERBASE":
+                "/env",  # this is adding the deps
+                "PATH":
+                "/env/bin:/opt/python%s/bin:/opt/python3.5/bin:" % py_version +
```

Comment:
I think you might be able to use $PATH here instead of hardcoding the existing path elements.

Reply:
it wasn't clear to me as this is an override if this is called with context to what the PATH variable is currently